data/human/fleets.txt

```diff
@@ -3488,3 +3488,105 @@ fleet "Human Wormhole Guards"
 		"Frigate (Mark II)"
 	variant 5
 		"Rainmaker (Mark II Old Weapons)"
+
+fleet "Derelict Republic"
+	government "Derelict (Boardable)"
+	names "civilian"
+	"cargo settings"
+		cargo 2
+		outfitters "Derelict Republic"
+	personality derelict mute marked
+	variant 18
+		"Scout (Derelict)"
+	variant 12
+		"Hauler (Derelict)"
+	variant 5
+		"Hauler II (Derelict)"
+	variant 18
+		"Berserker (Derelict)"
+	variant 15
+		"Freighter (Derelict)"
+	variant 2
+		"Star Queen (Derelict)"
+	variant 14
+		"Firebird (Derelict)"
+	variant 12
+		"Aerie (Derelict)"
+	variant 1
+		"Leviathan (Derelict)"
+	variant 20
+		"Sunder (Derelict)"
```

Comment:
I'd sort these by descending weight so that the highest spawn rates are at the top. That would make comparing spawn rates easier.